website/docs/signing.md

```diff
@@ -1,5 +1,7 @@
 # Signing Packages
 
+***Note:*** Available with Dalec release `v0.3.0` and later.
```

Comment:
```suggestion
:::note
Available with Dalec release `v0.3.0` and later.
:::
```

Reply:
This is a docusaurus specific syntax: https://docusaurus.io/docs/markdown-features/admonitions

Reply:
Oh, cool!